packages/cfd/src/Containers/cfd-compare-accounts/__tests__/cfd-compare-accounts-description.spec.tsx

```diff
@@ -26,12 +32,18 @@ describe('<CFDCompareAccountsDescription />', () => {
     };
 
     it('should render CFDCompareAccountsDescription component on default props', () => {
-        render(<CFDCompareAccountsDescription {...mocked_props} />);
+        const wrapper = ({ children }: { children: JSX.Element }) => (
```

Comment:
Define wrapper on the higher scope so that you do not need to redefine it in every test case